src/pkg/err/err.go

```diff
@@ -0,0 +1,45 @@
+package err
+
+import (
+	"errors"
+
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	"github.com/alcionai/corso/src/pkg/repository"
+)
+
+// expose enums, rather than errors, for Is checks.  The enum should
+// map to a specific internal error that can be used for the actual
+// errors.Is comparison.
+type errEnum string
+
+const (
+	RepoAlreadyExists errEnum = "repository-already-exists"
+	BackupNotFound    errEnum = "backup-not-found"
+	ServiceNotEnabled errEnum = "service-not-enabled"
+)
+
+// map of enums to errors.  We might want to re-use an enum for multiple
+// internal errors (ex: "ServiceNotEnabled" may exist in both graph and
+// non-graph producers).
+var internalToExternal = map[errEnum][]error{
```

Comment:
we're going to need some process for corso devs to make sure this stays up to date

Reply:
Lets bring it up as a topic in the monday meeting.